File: app/src/main/java/com/github2136/mvp/model/UserModel.java
package com.github2136.mvp.model;

import android.content.Context;

import com.github2136.base.BaseMVPModel;

import okhttp3.Callback;

/**
 *
  **/
public class UserModel extends BaseMVPModel {

    public UserModel(Context context, String tag) {
        super(context, tag);
    }

    public void get(Callback callback) {
        httpGet("http://www.asfrvre.com", "", null, callback);
    }
}

File: app/src/main/java/com/github2136/mvp/model/UserModel.kt
package com.github2136.mvp.model

import android.app.Application
import com.github2136.base.BaseMVPModel
import okhttp3.Callback

/**
 * Created by yb on 2018/11/2.
 */
class UserModel(app: Application, tag: String) : BaseMVPModel(app, tag) {

    fun get(callback: Callback) {
        httpGet("http://www.baidu.com", "", null, callback)
    }
}